### PR TITLE
Mijn 667 responsive header footer

### PR DESCRIPTION
**Pull request by BennievanderWel**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ function MainApp({ appState: { SESSION, BRP } }: MainAppProps) {
   return location.pathname === AppRoutes.MY_AREA ? (
     <MyArea />
   ) : (
-    <>
+    <div className={styles.App}>
```
[the remark this answers is not in the document]
Comment (BennievanderWel) [reply src/App.tsx]: Done!

File: src/components/MainFooter/MainFooter.tsx [hunk 2]
```diff
@@ -1,18 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styles from './MainFooter.module.scss';
 import ButtonLink, {
   ButtonLinkExternal,
 } from 'components/ButtonLink/ButtonLink';
 import { ExternalUrls, AppRoutes } from 'App.constants';
 import { LinkList } from './MainFooter.constants';
 import classnames from 'classnames';
+import { isLargeScreen } from 'helpers/Media';
```
[the remark this answers is not in the document]
Comment (BennievanderWel) [reply src/components/MainFooter/MainFooter.tsx]: Agreed and processed.

File: src/components/MainHeader/MainHeader.module.scss [hunk 3]
```diff
@@ -102,6 +123,15 @@
       padding-top: 1rem;
       border-top: 0.1rem solid $color-neutral-grey2;
       height: 4rem;
+
+      // Target a specific anchor with high CSS specificity
```
[the remark this answers is not in the document]
Comment (BennievanderWel) [reply src/components/MainHeader/MainHeader.module.scss]: OK

File: src/components/MainFooter/MainFooter.tsx [hunk 4]
```diff
@@ -1,18 +1,38 @@
...
 
 export default function MainFooter() {
+  const [panelStates, setPanelState] = useState({} as any);
```
[the remark this answers is not in the document]
Comment (timvanoostrom) [reply src/components/MainFooter/MainFooter.tsx]:
```
interface PanelState {
  [panelId: string]: boolean
}
```

**Comment by timvanoostrom** [comment]
LGTM

**Pull request merged**